internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerClient.java

```diff
@@ -732,8 +733,12 @@ public MultiNodeResponse listStorageNodes() {
     return request(ControllerRoute.LIST_NODES, newParams(), MultiNodeResponse.class);
   }
 
-  public MultiNodesStatusResponse listInstancesStatuses() {
-    return request(ControllerRoute.ClUSTER_HEALTH_INSTANCES, newParams(), MultiNodesStatusResponse.class);
+  public MultiNodesStatusResponse listInstancesStatuses(boolean enableReplicas) {
+    QueryParams params = newParams();
+    if (enableReplicas) {
+      params.add(ENABLE_DISABLED_REPLICAS, "true");
```

Comment:
IMO, enabling error replicas in a command which was previously used for listing only, can be risky as it can have side-effect that operator may not be aware of.
Can we also update the [instruction](https://github.com/linkedin/venice/blob/f45c164792ae9877a3ee12038cc6d56b8e764162/clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java#L170) here as well so people get informed when they use this command?

Reply:
Its an extra option to the existing command. So operator wont know unless they want to use it.